server/datastore/mysql/labels.go

```diff
@@ -370,14 +370,8 @@ func (d *Datastore) RecordLabelQueryExecutions(ctx context.Context, host *fleet.
 		err := d.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
 			// Complete inserts if necessary
 			if len(vals) > 0 {
-				sql := `
-        INSERT IGNORE INTO label_membership (updated_at, label_id, host_id) VALUES
-        `
-				sql += strings.Join(bindvars, ",") +
-					`
-          ON DUPLICATE KEY UPDATE
-          updated_at = VALUES(updated_at)
-        `
+				sql := `INSERT IGNORE INTO label_membership (updated_at, label_id, host_id) VALUES `
+				sql += strings.Join(bindvars, ",") + ` ON DUPLICATE KEY UPDATE updated_at = VALUES(updated_at)`
```

Comment:
The `IGNORE` could be removed given that the `ON DUPLICATE KEY UPDATE` clause is there.

Reply:
Ah right, we added that in case a label_id wasn't there anymore, but that doesn't matter.